google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQueryOptions.java

```diff
@@ -61,6 +64,15 @@ private Builder(BigQueryOptions options) {
     }
 
     @Override
+    public Builder setTransportOptions(TransportOptions transportOptions) {
```

Comment:
@michaelbausor We probably want to allow the setting of transport options in `ServiceOptions`.